target/linux/x86/image/Makefile

```diff
@@ -217,14 +325,22 @@ define Image/Build
 	$(call Image/Build/$(1))
   ifneq ($(1),iso)
 	$(call Image/Build/grub2,$(1))
+	$(call Image/Build/efi,$(1))
```

Comment:
Yes, the squashfs image boot correctly with EFI because the kernel partition is in ext4, only the rootfs partition is in squashfs (because squashfs is read only with an overlay in jffs2).